src/main/java/de/tum/in/www1/artemis/web/rest/tutorialgroups/TutorialGroupResource.java

```diff
@@ -161,7 +162,7 @@ public ResponseEntity<Set<String>> getUniqueLanguageValues(@PathVariable Long co
     public ResponseEntity<List<TutorialGroup>> getAllForCourse(@PathVariable Long courseId) {
         log.debug("REST request to get all tutorial groups of course with id: {}", courseId);
         var course = courseRepository.findByIdElseThrow(courseId);
-        var user = userRepository.getUserWithGroupsAndAuthorities();
+        var user = userRepository.getUser();
```

Comment:
The groups are needed in the next call (checkHasAtLeastRoleInCourseElseThrow), which currently leads to an additional database request.